tests/fsstress/testdata/fsstress_test.txt

```diff
@@ -0,0 +1,31 @@
+{{$test_opts := "-test.v -name fsstress-app"}}
+
+# Starting of reboot detector with a 1 reboot limit
+! test eden.reboot.test -test.v -timewait 45m -reboot=0 -count=1 &
+
+#TestFSstressVMStart checks if app processed by EVE, app in RUNNING state
+test eden.fsstress.test {{$test_opts}} -timewait 15m -test.run TestFSstressVMStart
+stdout '--- PASS: TestFSstressVMStart'
+
+# TestAccess checks if SSH of app is accessible
+test eden.fsstress.test {{$test_opts}} -timewait 10m -test.run TestAccess
+stdout '--- PASS: TestAccess'
+
+# TestRunStres checks if SSH of app is accessible
+test eden.fsstress.test {{$test_opts}} -timewait 96h -test.run TestRunStress
```

Comment:
Do we really expect the end of the test to come in 96h? Or we must stop it somehow?

Reply:
that's just a magic number for now. We are trying to create some building blocks for now, just a machinery to run tests for really long period of time (e.g. 7 days). But at this point 96h is not important

Reply:
Actually my question mostly about an expectation. Do we expect that test will indicates PASS after our long waiting? If so, seems, we need to break our run-script.sh somehow, elsewhere we will face timeout in any cases.

Reply:
In our case, the success of the test depends on the lifetime, if I/O on the guest OS works after the timer expires, then the test was successful. If I / O does not work, it most likely means I/O problems have and in EVE, which means all will be broken.

Reply:
Well can we detect the error faster?  Maybe I miss something... Also is it ok to use the go test mechanism waiting for so long?

Reply:
A good question аbout waiting, but what can stop us?
This test is required to test storage fault tolerance in EVE. If an error occurs in the test earlier than n-timewait, then the whole EVE will most likely crash. That is, when and if EVE breaks, it will be a negative test result. And it is important for us to live the whole test. 

I also have concerns that after such a long period, we may wait a long time for a response from the VM (due to the peculiarities of Eden), which may also affect its duration and correct execution.